src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/helpers/terraform/get_accounts.py

```diff
@@ -41,10 +41,7 @@ def main():
 def list_organizational_units_for_parent(parent_ou):
     organizations = get_boto3_client(
         'organizations',
-        (
-            f'arn:{PARTITION}:sts::{MANAGEMENT_ACCOUNT_ID}:role/'
-            f'{CROSS_ACCOUNT_ACCESS_ROLE}-readonly'
-        ),
+        f'arn:{PARTITION}:sts::{MANAGEMENT_ACCOUNT_ID}:role/{CROSS_ACCOUNT_ACCESS_ROLE}-readonly',
```

Comment:
Could you please undo these changes?
These strings were broken up in two to match the line length requirements as set in the [editorconfig](https://github.com/awslabs/aws-deployment-framework/blob/master/.editorconfig).

The result should be exactly the same string.

Reply:
Rollback and tested successfully